manifests/certonly.pp

```diff
@@ -33,18 +33,19 @@
 #   succeeds.
 #
 define letsencrypt::certonly (
-  Array[String[1]]        $domains              = [$title],
-  Boolean                 $custom_plugin        = false,
-  Letsencrypt::Plugin     $plugin               = 'standalone',
-  Array[Stdlib::Unixpath] $webroot_paths        = [],
-  String[1]               $letsencrypt_command  = $letsencrypt::command,
-  Array[String[1]]        $additional_args      = [],
-  Array[String[1]]        $environment          = [],
-  Boolean                 $manage_cron          = false,
-  Boolean                 $suppress_cron_output = false,
-  Optional[String[1]]     $cron_before_command  = undef,
-  Optional[String[1]]     $cron_success_command = undef,
-  Stdlib::Unixpath        $config_dir           = $letsencrypt::config_dir,
+  Array[String[1]]                   $domains              = [$title],
+  Boolean                            $custom_plugin        = false,
+  Letsencrypt::Plugin                $plugin               = 'standalone',
+  Array[Stdlib::Unixpath]            $webroot_paths        = [],
+  String[1]                          $letsencrypt_command  = $letsencrypt::command,
+  Array[String[1]]                   $additional_args      = [],
+  Array[String[1]]                   $environment          = [],
+  Boolean                            $manage_cron          = false,
+  Boolean                            $suppress_cron_output = false,
+  Optional[String[1]]                $cron_before_command  = undef,
+  Optional[String[1]]                $cron_success_command = undef,
+  Array[Variant[Integer, String[1]]] $cron_monthday        = ['*'],
```

Comment:
Is there a known minimal/maximal value for the integer? That can also be enforced with:
```suggestion
  Array[Variant[Integer[0, 60], String[1]]] $cron_monthday = ['*'],
```

See also: https://puppet.com/docs/puppet/5.4/lang_data_number.html#the-integer-data-type

Reply:
Well spotted! Based on the man page of crontab the integer value can be between 0-59. I will commit the change to this branch in a few minutes.

Reply:
thanks. I will merge it after it turned green.

Reply:
Cheers :+1: